src/ui/deleteAccount/ViewDeleteAccount.qml

```diff
@@ -17,12 +17,12 @@ Item {
 
     Loader {
         id: loader
-        property bool isDeleteAccountAuth: true
+        property bool isReauthFlow: true
 
         anchors.fill: parent
```

Comment:
I talked back with UX and we decided to leave show the re-authentication flow as implemented. I’d like to address general updates and fixes as a follow up with _Consolidate authentication flows_ ([VPN-2145](https://mozilla-hub.atlassian.net/jira/software/c/projects/VPN/boards/344?modal=detail&selectedIssue=VPN-2145)).

Reply:
I talked back with UX and we decided we decided on not overlaying the view. I’d like to address general updates and changes together with [VPN-2145 ](https://mozilla-hub.atlassian.net/jira/software/c/projects/VPN/boards/344?modal=detail&selectedIssue=VPN-2145)in order to not add any more complexity to this PR. Happy to talk about it in more detail if there if we should revisit this.

Reply:
The re-authentication template is currently build in a way that it can be used with any view as a destination. It can be provided with a view that should be shown after the re-auth flow is completed and it could be used anywhere. Happy to tweak the behaviour, but I think it might make sense to talk about it a bit more.